src/lobsterpy/cli.py

```diff
@@ -561,7 +561,7 @@ def get_parser() -> argparse.ArgumentParser:
         ],
         help=(
             "Deliver a text description of the LOBSTER calc quality analysis. "
-            "Mandatory required files: POSCAR, POTCAR, lobsterout, lobsterin. "
+            "Mandatory required files: CONTCAR, POTCAR, lobsterout, lobsterin. "
```

Comment:
Would change it to "CONTCAR or other structure file compatible with --file-structure"